deployment/aws/lambda/Dockerfile

```diff
@@ -4,19 +4,32 @@ WORKDIR /tmp
 
 COPY titiler.*.tar.gz /tmp/
 
-# rasterio 1.2.0 wheels are built using GDAL 3.2 and PROJ 7 which we found having a
-# performance downgrade: https://github.com/developmentseed/titiler/discussions/216
 RUN pip install \
+        # mangum is required by our handler.py, and isn't a dependency of the
+        # titiler packages
+        mangum==0.17.0 \
+        # pin rasterio, morecantile, rio-tiler, and
+        # cogeo-mosaic to fix issue with openssl initialization failing, 
+        # causing 500 responses. Rasterio <1.3 seemed to be the culprit for that,
+        # and rasterio 1.3 has better packages for curl, openssl, and/or gdal
+        rasterio==1.3.6 \
+        morecantile==3.2.5 \
+        rio-tiler==4.1.9 \
+        cogeo-mosaic==5.1.1 \
+        # pin markupsafe so jinja will have access to soft_unicode
+        # https://github.com/aws/aws-sam-cli/issues/3661#issuecomment-1044340547
+        markupsafe==2.0.1 \
         titiler.application-0.3.5.tar.gz titiler.mosaic-0.3.5.tar.gz titiler.core-0.3.5.tar.gz \
-        mangum>=0.10.0 rasterio==1.1.8 -t /var/task --no-binary numpy,pydantic --no-cache-dir
+        -t /var/task --no-binary pydantic
+        # original code also had --no-binary numpy, which takes 20-30 minutes to build
```

Comment:
We want to keep this -- pinning these made the build run a lot faster.

I wonder if they changed the dependencies so that application always depends on mosaic and core, or if it was always redundant?